src/Enums.jl

```diff
@@ -4,9 +4,13 @@ module Enums
     using FixedPointDecimals
     using ..ChunkedCSV: GuessDateTime
 
+    # Enums used to represent known types that we manually unroll in populate_result_buffer.jl
+    # Unrolling on enums is easier for the compiler than unrolling on types. For unknown types,
+    # we use a generated function to unroll on the types in the schema, see `parsecustom!` in
+    # src/row_parsing.jl for how this is used.
```

Comment:
there is no `src/row_parsing.jl`?

also i was confused by the `types` in this sentence the first time i read it. how about:
```suggestion
    # Enums used to represent _known_ input-types that we manually unroll in populate_result_buffer.jl
    # Unrolling on enums is easier for the compiler than unrolling on types. 
    # For _unknown_ input-types, we use a generated function to unroll on the types in the schema, 
    # see `parsecustom!` for how this is used.
```

Reply:
Ah, `row_parsing.jl` was renamed to `populate_result_buffer.jl`